api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -269,6 +297,39 @@ private WorkspaceExportRow toWorkspaceExportRow(DbUser user) {
             .map(dateFormat::format)
             .orElse(""));
     row.setCreatorRegistrationState(user.getDataAccessLevelEnum().toString());
+    row.setDegrees(
+        Optional.ofNullable(user.getDegreesEnum()).orElse(ImmutableList.of()).stream()
+            .map(Degree::toString)
+            .collect(Collectors.joining(",")));
```

Comment:
nit: the rest of the multi value columns are separated with a newline as well

Reply:
Thanks, missed that.